### DISCUSSION TITLE
React-Select with React-hook form get the value without label

### DISCUSSION THREAD

**Discussion by szelam** [react-hook-form/react-hook-form#2448] [Q&A] [answered]
How can I get the output without label:"Apple", I only want the value.

now :**{ value: "1", label: "Apple"}**
target: **1**

here is my code [https://codesandbox.io/s/controller-with-react-select-qz78k?file=/src/index.js](https://codesandbox.io/s/controller-with-react-select-qz78k?file=/src/index.js)

Please Help....
Appreciate for any answer !!!!

**Comment by Keony1** [comment, marked as answer]
I think in this case you have 2 options:

1 - Wrap react-select in one component and in onChange of react-select when action "select-option" is fired, you just return the value of the option selected. something like this:

![image](https://user-images.githubusercontent.com/30664147/89110290-af661900-d41f-11ea-8d8a-054e2b980cbc.png)

2 - Or you handle it inside formsubmit

**Reply by szelam** [reply]
How about the second options? I don’t know how to handle it

**Reply by Keony1** [reply]
you can do this

![image](https://user-images.githubusercontent.com/30664147/89123603-f77c4e80-d4a6-11ea-990a-1b529a3c1f39.png)

**Reply by szelam** [reply]
Thank you very much!

**Reply by bluebill1049** [reply]
you can read from the `data` object as well :)
